walnut/converter/converter.go

```diff
@@ -111,13 +85,58 @@ func ExtractTaskClasses(dplDump Dump) (tasks []*task.Class, err error) {
 				Memory: createFloat(128),
 				Ports:  task.Ranges{}, //begin - end OR range
 			},
-			Bind: bind,
 			Properties: gera.MakeStringMapWithMap(map[string]string{
 				"severity": "trace",
 				"color":    "false",
 			}),
-			Connect: connect,
 		}
+
+		module = append(module, "Control-OCCPlugin")
```

Comment:
I'd prefer if `Control-OCCPlugin` was always the last module in the slice.

Reply:
Is that not what's happening here? `module` is a `[]string` provided to this function. Whether it's empty or not, `Control-OCCPlugin` is appended to it.

Reply:
You're right, my bad.